core/src/com/biglybt/core/networkmanager/impl/tcp/SelectorGuard.java

```diff
@@ -117,8 +117,9 @@ public void verifySelectorIntegrity( int num_keys_ready, long time_threshold ) {
         //Under windows, it seems that selector spin can sometimes appear when >63 socket channels are registered with a selector
```

Comment:
This relic if-block is always skipped and should be removed.

The implications of removing it is removing the only usage of 
`listener.safeModeSelectEnabled()` and `listener.spinDetected()`.
@parg, this is your domain. Please advice.

Reply:
Yup, looks that way

Reply:
I opened #1005 for this